src/components/actions/filters/ItemFilterActions.ts

```diff
@@ -0,0 +1,37 @@
+import { IReduxAction } from '../../../utils/ReduxUtils';
+
+export const ItemFilterActions = {
+  add: 'ADD_ITEM_FILTER',
+  filter: 'FILTER_ITEMS',
+  remove: 'REMOVE_ITEM_FILTER'
+};
+
+export interface IItemFilterActionPayload {
```

Comment:
I think it's better to keep them separated since we might want to change one but not the others.